index.html

```diff
@@ -3071,13 +3088,26 @@ <h5>Supported use cases</h5>
 
 <h5>Uses beyond mapping</h5>
 <p>
-<!-- What other web content could use this specific capability? Be brief but specific. -->
+This functionality can be used to provide hyperlink support for diagrams.
+Any case where an image map may have been appropriate can also benefit from this capability.
 </p>
 
 <h5>Related web specifications</h5>
 <p>
-<!-- Any existing/proposed specs that address similar needs or otherwise interact.
-      Include links with proper ReSpec references. -->
+<a data-cite="HTML#image-maps">HTML Image Maps</a> consist of a large graphic with various subregions,
+which serve as links.
+A subregion is defined by passing a shape, coordinates, a link target and alt text to an <code>area</code> element.
+A significant limitation of Image Maps is that they are not responsive by default,
```

Comment:
@AmeliaBR Perhaps this should link to https://github.com/whatwg/html/issues/389?